indra/sources/signor.py

```diff
@@ -22,9 +22,27 @@
 from indra.util import read_unicode_csv, read_unicode_csv_fileobj
 from indra.databases import hgnc_client, uniprot_client
 
-
 logger = logging.getLogger('signor')
 
+def _read_famplex_map():
+    fname = os.path.join(dirname(__file__), '../resources/famplex_map.tsv')
+    raw_map = read_unicode_csv(fname, '\t')
+
+    m = {}
+    for row in raw_map:
+        m[(row[0], row[1])] = row[2]
+    return m
+famplex_map = _read_famplex_map()
+
+def _read_signor_complex_map():
```

Comment:
do you think that you could set this up to pull the latest set of complexes from the SIGNOR website, rather than a file added to /resources? The SIGNOR processor could make a post request to the SIGNOR website for this data (see how the interaction data is downloaded).